libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -379,6 +403,19 @@ export type TigerSpecificFunctions = {
     ) => Promise<IInvitationUserResponse>;
     getWorkspaceDataFiltersLayout?: () => Promise<WorkspaceDataFiltersLayout>;
     setWorkspaceDataFiltersLayout?: (workspaceDataFiltersLayout: WorkspaceDataFiltersLayout) => Promise<void>;
+    getWorkspaceDataFilters?: (workspaceId: string, objectId: string) => Promise<WorkspaceDataFiltersResult>;
+    setWorkspaceDataFilters?: (
```

Comment:
Please rename `setWorkspaceDataFilters` → `setWorkspaceDataFilter` (without any `s` after `Filter`).

`setWorkspaceDataFilters` can make people think that the function can set an array of workspace data filters, can't it?